APCAppCore/APCAppCore/Library/Scheduler/APCScheduleDebugPrinter.m

```diff
@@ -249,6 +264,23 @@ + (NSString *) stringFromDate: (NSDate *) date
     return result;
 }
 
+- (NSString *) stringWithMillisecondsFromDate: (NSDate *) date
```

Comment:
I'll certainly change it if you feel strongly about it.  My thoughts:  The debugPrinter class also has a `-stringFromDate:` method, which prints a lower-resolution time stamp.  I've used `-stringFromDate:` in a bunch of places, inside and outside the class.  In general, when I'm writing, I navigate the code using Xcode's context menu, and I'll name classes and methods so they alphabetize together in the pop-up menu.  So, here, when I start to type the name of the method, like `[printer str...`, I've found it helpful to have that pop-up remind me I've got 2 versions of the method -- one high-res and one low-res.